urfit-tech/lodestar-app-admin
# Feat/display header footer option

# Add Program intro page display header and footer option
## how to test 
1. navigate to [program setting page ](http://localhost:5555/admin/programs/01c21a07-02ac-4854-a45e-45b9ec509859?tab=general)
2. update the program header and footer display settings and check the program table whether update is correct or not 
3. db table: program / column: display_header & display_footer
<img width="296" alt="image" src="https://github.com/urfit-tech/lodestar-app-admin/assets/107536598/9afa91bc-6267-40d9-8480-f8122767b716">
